code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -3,6 +3,10 @@
 	singular_name = "nanite swarm"
 	desc = "A tube of paste containing swarms of repair nanites. Very effective in repairing robotic machinery."
 	icon = 'icons/obj/stacks/nanopaste.dmi'
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/stacks/lefthand_nanopaste.dmi',
+		slot_r_hand_str = 'icons/mob/items/stacks/righthand_nanopaste.dmi',
+		)
```

Comment:
Why isn't this just in the medical icons? :thinking:

Reply:
It doesn't have the /medical/ suffix

also it's used outside of medical anyway